### DISCUSSION TITLE
How to open mce software?

### DISCUSSION THREAD

**Discussion by AhmedHalool** [supratimdasinfo/magnetocaloric.mcepy#3] [Q&A] [answered]
Dear Administrators,
First i want to thank you for this great  job. Actually, i want to use the mcepy software to plot my data, i followed the instructions for installation as follow: 
1. I downloaded python for windows from  Anconda.Org
2. I opened Anconda  Navigator then  I Launched the Jupyter lab  
3. I wrote the the installing code [**pip install magnetocaloric==1.7.1**]
**it was downloaded successfully**
until this step i stopped and i can not continue  to open the mcepy software as you used it in video.......?
So please give me the in details steps to open the software  { such as codes }
I see you videos on how to use the software but i didn't open it mainly.
Thanks in Advance and looking forward to hear from you.
Appreciate your response,
Sincerely,
Ahmed Halool
Ph.D Researcher

**Comment by supratimdasinfo** [comment]
Good to see your interest in this project. I hope this reply finds you well.

The module 'magnetocaloric.mcepy' is freely available to everyone. Users can install it using pip and perform calculations via the command line interface. However, the GUI you might have seen on YouTube will not appear there, as it is only available to a limited number of users.

For more information or queries, feel free to reach out to me via email.

**Comment by AhmedHalool** [comment, marked as answer]
Thank you for your clarification. Lastly,I want to know how to get the GUI
that you used it in YouTube video.  I need the access please.

Ahmed Halool
Assistant lecturer
Damietta University, Faculty of Science, Physice Department
Theoretical Physics
Tel: +201004906899
ZIP code: 34517

**Reply by supratimdasinfo** [reply]
Respected sir,
There's few terms and conditions, For further negotiations mail to supratimdas.connect@gmail.com
